reference/core/ingress-controller.md

```diff
@@ -255,7 +255,7 @@ Read more from Kubernetes [here](https://kubernetes.io/docs/concepts/services-ne
 ### TLS Termination
 
 ```yaml
-apiVersion: extensions/v1beta1
+apiVersion: extensions/v1
```

Comment:
This sample is out of date with the `v1` API.
@alexgervais to provide a good and valid example.